aplib.net-demo/Assets/Scripts/Entities/Player/Movement.cs

```diff
@@ -34,13 +38,20 @@ public class Movement : MonoBehaviour
     [SerializeField]
     private Transform _playerVisTransform;
 
+    private EntitySound _footStep;
+    private Timer _timer;
+
     /// <summary>
     /// Initialize the player's rigidbody and collider, and freeze the player's rotation.
     /// </summary>
     private void Start()
     {
+        // Link the components of this object to the variables
```

Comment:
Why not put this in the summary?

Reply:
Because it is a comment on that block of code and not the entire function, wanted to group it and then comment it for clarity.